src/dmd/root/response.d

```diff
@@ -16,7 +16,6 @@ module dmd.root.response;
 
 import dmd.root.file;
 import dmd.root.filename;
-import dmd.utils;
```

Comment:
It's used in a `version (unittest)`

Reply:
Oh. :-)

Reply:
Fixed (moved the import inside the unittest).